project/plugins.sbt

```diff
@@ -2,7 +2,8 @@ resolvers += Resolver.url("HMRC Sbt Plugin Releases", url("https://dl.bintray.co
 
 resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
 
-resolvers += "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
+resolvers += ("Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/").withAllowInsecureProtocol(true)
```

Comment:
Why have we set withAllowInsecureProtocol to true here?

Reply:
I think with the sbt 1.3 upgrade we get loads of HTTP warns in terminal and this disables them. Im not 100% sure about this however and its probably worth bouncing off Amaal on this one.